.gitignore

```diff
@@ -0,0 +1,2 @@
+/cache/*
```

Comment:
please alter the Pull Request so git doesn't ignore the ``cache/installedVersion.json`` and ``content/.gitkeep``
than I will merge it.